test/functional/tool_wallet.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
```

Comment:
Add this file into test_runner.py script list?

Reply:
done